src/views/LatestMovieView.vue

```diff
@@ -1,16 +1,14 @@
 <template>
-  <MoviesList :movies="latestMovie"/>
+  <div>
```

Comment:
add CSS class to the root element:
`class="latest-movies"`

Reply:
I remove MoviesList component, becouse of endpint of latestMovie has data for one film.